<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>555</width>
    <height>590</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>555</width>
    <height>590</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>555</width>
    <height>590</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>401</width>
      <height>521</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBoxMain">
       <property name="title">
        <string>Subtitle Extractor</string>
       </property>
       <widget class="QWidget" name="formLayoutWidget">
        <property name="geometry">
         <rect>
          <x>8</x>
          <y>20</y>
          <width>381</width>
          <height>501</height>
         </rect>
        </property>
        <layout class="QFormLayout" name="formLayout_2">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="labelFiles">
           <property name="text">
            <string>Files</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QListWidget" name="listWidgetFiles">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>260</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Select files hit delete to remove from list.</string>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::MultiSelection</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="labelSubDirectory">
           <property name="toolTip">
            <string>Put .SUP files in a different directory</string>
           </property>
           <property name="text">
            <string>Subtitle Sub
Directory</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lineEditSubDirectory">
           <property name="toolTip">
            <string>For on directory, just type it alone. For nested, add forward slash between them.</string>
           </property>
           <property name="placeholderText">
            <string>(Leave blank to place subtitles in same directory)</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label">
           <property name="toolTip">
            <string>Use MPHC's properties&gt;media info to find subtitle track IDs</string>
           </property>
           <property name="text">
            <string>Stream ID #s</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="lineEditTrackID">
           <property name="placeholderText">
            <string>(delimit with commas for multiple subs; ex: &quot;4, 5, 6&quot;)</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="labelMKV">
           <property name="text">
            <string>MKV Address</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="lineMKVAddress">
           <property name="toolTip">
            <string>Add your mkvextract.exe's address to your System Environment Variables PATH.
BE CAREFUL. DO NOT ERASE ANYTHING THAT IS ALREADY THERE.</string>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="placeholderText">
            <string>(if not set, will attempt to use your PATH variable)</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>555</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSelect_File"/>
    <addaction name="actionSelect_Directory"/>
    <addaction name="separator"/>
    <addaction name="actionDelete_File"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSelect_File"/>
   <addaction name="actionSelect_Directory"/>
   <addaction name="actionFind_MKVExtract_exe"/>
   <addaction name="actionRip"/>
  </widget>
  <action name="actionSelect_Directory">
   <property name="text">
    <string>Select Directory</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+O</string>
   </property>
  </action>
  <action name="actionSelect_File">
   <property name="text">
    <string>Select File(s)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionDelete_File">
   <property name="text">
    <string>Delete File</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionFind_MKVExtract_exe">
   <property name="text">
    <string>Find MKVExtract.exe</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionRip">
   <property name="text">
    <string>Rip!</string>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
